draft-ietf-rtcweb-jsep.xml

```diff
@@ -1433,7 +1433,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           PeerConnection to apply the supplied session description as
           its local configuration. The type field indicates whether the
           description should be processed as an offer, provisional
-          answer, or final answer; offers and answers are checked
+          answer, rollback, or final answer; offers and answers are checked
```

Comment:
I think I would put 'rollback' at the end, and then say 'the SDP for offers and answers is checked differently, using ... that exist for each line.'